BedrockPlugin.cpp

```diff
@@ -34,6 +34,20 @@ void BedrockPlugin::verifyAttributeSize(const SData& request, const string& name
     }
 }
 
+BedrockPlugin* BedrockPlugin::getPlugin(const string& name) {
+    // Just loop over and return, if any
+    BedrockPlugin* foundPlugin = 0;
+    for_each(BedrockPlugin::g_registeredPluginList->begin(), BedrockPlugin::g_registeredPluginList->end(),
+             [&](BedrockPlugin* plugin) {
+                 if (SIEquals(plugin->getName(), name)) {
+                     foundPlugin = plugin;
```

Comment:
This is an annoying limitation in `std::for_each`. The performance penalty here is pretty minimal, though.